types/sauronjs/util/ready.d.ts

```diff
@@ -0,0 +1 @@
+export function ready(fn: () => void): void;
```

Comment:
Same thing here as my previous comment - this is a callable module.

Reply:
Done.